src/animation/AnimationObjectGroup.js

```diff
@@ -377,7 +375,8 @@ Object.assign( AnimationObjectGroup.prototype, {
 
 	}
 
-} );
+}
 
+AnimationObjectGroup.prototype.isAnimationObjectGroup = true;
```

Comment:
could you use the `Object.defineProperty()` pattern demonstrated in #20110 across the files modified in this PR? 🙏

Also support for class fields is coming soon.

Reply:
Will do 👍

Reply:
Fixed in c4872f5... thanks!